qemu/tests/vioser_in_use.py

```diff
@@ -116,10 +133,20 @@ def run(test, params, env):
 
     bg_thread = run_bg_test(test, params, vm, sender)
     globals().get(params["interrupt_test"])(test, params, vm, session)
-    if bg_thread:
-        bg_thread.join(timeout=timeout, suppress_exception=suppress_exception)
-    if vm.is_alive():
-        kill_host_serial_pid(params, vm)
-        if (virtio_serial_file_transfer.transfer_data(params, vm, sender=sender) is
-                not True):
-            test.fail("Serial data transfter test failed.")
+
+    # for vcpu hotplug subtest, only check guest crash.
+    vcpu_hotplug = params.get_boolean("vcpu_hotplug")
```

Comment:
If the default bool of `vcpu_hotplug` is false, then there should be `vcpu_hotplug = params.get_boolean("vcpu_hotplug"，False)` .

Reply:
Since the default bool value is false it's fine with this way.